avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -8,10 +8,10 @@
 com1Exe = ./com1DFA.exe -files files/AK_Attributes/
 # True if full output shall be printed to terminal
 flagOut = False
-# True if entrainment areas shall be included
-flagEnt = True
-# True if resistance areas shall be included
-flagRes = True
+# True if entrainment areas shall NOT be included
+flagNoEnt = True
+# True if resistance areas shall NOT be included
+flagNoRes = True
```

Comment:
I think they should be `False` by default:
This leads to the expected behaviour of including entrainment if an entrainment shapefile is present, and not including it if none is present...

```suggestion
# True if entrainment areas shall NOT be included
flagNoEnt = False
# True if resistance areas shall NOT be included
flagNoRes = False
```